pkg/costmodel/cluster.go

```diff
@@ -775,10 +770,7 @@ func (a *Accesses) ComputeClusterCosts(client prometheus.Client, provider cloud.
 		queryTotalLocalStorage = fmt.Sprintf(" + %s", queryTotalLocalStorage)
 	}
 
-	fmtOffset := ""
-	if offset != "" {
-		fmtOffset = fmt.Sprintf("offset %s", offset)
-	}
+	fmtOffset := timeutil.DurationToPromString(offset)
```

Comment:
This looks suspicious, unless `timeutil.DurationToPromString` happens to add the string "offset " to the given duration, which would be unexpected.